.github/workflows/shared.yml

```diff
@@ -0,0 +1,49 @@
+name: OQS-BoringSSL (Shared)
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+      - 'oqs_template/**'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'oqs_template/**'
+  schedule:
```

Comment:
Same question as for demos: Why a scheduled build?

Reply:
to detect issues early, as updates to dependencies such as libunwind and Go can result in build failures.